tests/span/conftest.py

```diff
@@ -0,0 +1,145 @@
+'''
+Conftest file for span tests
+'''
+
+import pytest
+
+@pytest.fixture(scope="module")
+def cfg_facts(duthosts, rand_one_dut_hostname):
+    '''
+    Used to get config facts for selected DUT
+
+    Args:
+        duthosts: All DUTs belonging to the testbed.
+        rand_one_dut_hostname: hostname of a random chosen dut to run test.
+    '''
+    duthost = duthosts[rand_one_dut_hostname]
+    return duthost.config_facts(host=duthost.hostname, source="persistent")['ansible_facts']
+
+@pytest.fixture(scope="module")
+def ports_for_test(cfg_facts):
```

Comment:
I get removing the port from the VLAN if it is a VLAN member, but why do we need to explicitly choose 3 VLAN members as candidate ports for the test?

Reply:
I think the 3 ports are for:
```
        'source1': port_info[0],
        'source2': port_info[1],
        'monitor': port_info[2],
```